sys/malloc.c

```diff
@@ -267,3 +267,5 @@ void kmem_dump(kmem_pool_t *mp) {
 /* TODO: missing implementation */
 void kmem_destroy(kmem_pool_t *mp) {
 }
+
+MALLOC_DEFINE(M_TEMP, "temporaries pool", 2, 4);
```

Comment:
I like idea behind this pool. Going to be easier to implement some tests in other branches where I needed to perform some memory allocation. It wouldn't hurt to increase it's size to be even bigger (say 32 pages).